onnxscript/evaluator.py

```diff
@@ -115,59 +113,74 @@ class Evaluator(abc.ABC):
     supported by onnxscript to those expected by a particular backend.
     """
 
-    def eval(self, schema, inputs, attributes):
-        closure = self.adapt_attributes(schema, attributes)
+    def eval(
+        self, schema: onnx.defs.OpSchema, inputs: Sequence[Any], attributes: Mapping[str, Any]
```

Comment:
Is it clear that "Any" here means that we have not yet bothered to more precisely define the allowed types of values? As, opposed to asserting that any type of values can be passed? 

Concretely, would it make sense to introduce a type-alias that captures the intention, like
```py
InputValueType  = Any  # TODO: elaborate this.
```
and use it?

Reply:
Sounds good!

Reply:
I updated to use ExtendedModeValue